src/components/manage/Widgets/UrlWidget.jsx

```diff
@@ -70,7 +70,7 @@ export const UrlWidget = (props) => {
 
     setValue(newValue);
 
-    newValue = isInternalURL(newValue) ? addAppURL(newValue) : newValue;
+    newValue = isInternalURL(newValue) ? toPublicURL(newValue) : newValue;
```

Comment:
@dobri1408 Using this we might need to set publicUrl. Also, I think URL is transformed by the restapi when the widget is used as a metadata behavior.